xla/service/gpu/fusions/triton/triton_support_test.cc

```diff
@@ -737,7 +745,10 @@ using ReductionComputationTest =
 // computation and in regular HLO. See triton_support.cc for more details.
 TEST_P(ReductionComputationTest, DifferentBinaryOps) {
   auto [data_type, opcode, cc] = GetParam();
-  bool dtype_is_complex = data_type == C64 || data_type == C128;
+  if (opcode == HloOpcode::kAtan2 &&
```

Comment:
Same comment as in the `cbrt` PR, this check should be done in `DoesOpSupportType` (and same for the above filter)!